opentelemetry-api/tests/configuration/test_configuration.py

```diff
@@ -35,27 +38,33 @@ def test_singleton(self):
         {
             "OPENTELEMETRY_PYTHON_METER_PROVIDER": "meter_provider",
             "OPENTELEMETRY_PYTHON_TRACER_PROVIDER": "tracer_provider",
+            "OPENTELEMETRY_PYTHON_OThER": "other",
+            "OPENTELEMETRY_PTHON_TRACEX_PROVIDER": "tracex_provider",
```

Comment:
Something with a number in between is missing in this test.

Reply:
Added a test.